src/test/osd/osd-scrub-snaps.sh

```diff
@@ -92,7 +92,7 @@ function TEST_scrub_snaps() {
     rados -p $poolname rm obj4
     rados -p $poolname rm obj2
 
-    kill_daemons $dir KILL osd || return 1
+    kill_daemons $dir TERM osd || return 1
     sleep 5
```

Comment:
please remove `sleep 5`, because in kill_daemons function we would sleep to wait for the osd is really dead unless timeout. Others look good to me.